core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -178,54 +178,52 @@ var TopicEditorPage = function() {
   };
 
   this.publishTopic = async function() {
-    await publishTopicButton.click();
+    await action.click('Publish Topic Button', publishTopicButton);
     await waitFor.invisibilityOf(
       publishTopicButton, 'Topic is taking too long to publish.');
   };
 
   this.expectNumberOfQuestionsForSkillWithDescriptionToBe = async function(
       count, skillDescription) {
-    await waitFor.elementToBeClickable(
-      selectSkillDropdown, 'Skill select dropdown takes too long to appear.');
-    await selectSkillDropdown.click();
-    await element(by.css('option[label="' + skillDescription + '"]')).click();
+    await action.click('Select Skill Dropdown', selectSkillDropdown);
+    var skillDescriptionButton = element(
+      by.css('option[label="' + skillDescription + '"]'));
+    await action.click('Skill Description Button', skillDescriptionButton);
     await waitFor.visibilityOf(
       questionItems.first(), 'Question takes too long to appear');
     expect(await questionItems.count()).toEqual(count);
   };
 
   this.saveQuestion = async function() {
     await general.scrollToTop();
-    await saveQuestionButton.click();
+    await action.click('Save Question Button', saveQuestionButton);
     await waitFor.invisibilityOf(
       saveQuestionButton, 'Question modal takes too long to disappear');
   };
 
   this.createQuestionForSkillWithName = async function(skillDescription) {
     await action.click('Select skill dropdown', selectSkillDropdown);
-    await waitFor.elementToBeClickable(
-      selectSkillDropdown, 'Skill select dropdown takes too long to appear.');
-    await selectSkillDropdown.click();
-    await element(by.css('option[label="' + skillDescription + '"]')).click();
-
+    var skillDescriptionButton = element(
+      by.css('option[label="' + skillDescription + '"]'));
+    await action.click('Skill Description Button', skillDescriptionButton);
     await action.click('Create question button', createQuestionButton);
     await action.click('Easy difficulty for skill', easyRubricDifficulty);
   };
 
   this.moveToQuestionsTab = async function() {
-    await waitFor.elementToBeClickable(
-      questionsTabButton,
-      'Questions tab button takes too long to be clickable');
-    await questionsTabButton.click();
+    await action.click('Questions Tab Button', questionsTabButton);
   };
 
   this.expectSubtopicPageContentsToMatch = async function(contents) {
-    var text = await subtopicContentText.getText();
+    var text = await action.getText(
+      'Subtopic Content Text', subtopicContentText);
     expect(text).toMatch(contents);
   };
 
   this.expectTitleOfSubtopicWithIndexToMatch = async function(title, index) {
-    expect(await subtopics.get(index).getText()).toEqual(title);
+    var subtopic = subtopics.get(index);
+    var text = await action.getText('Subtopic Text', subtopic);
+    expect(text).toEqual(title);
   };
 
   this.changeSubtopicTitle = async function(title) {
```

Comment:
Hmm interesting. I'm not sure why you were seeing failures when you used the `action` functions here. Could you try adding `waitFor` statements to wait for `subtopicTitleField` to be visible? That will mimic what the `action` functions achieve, but hopefully without causing errors. Please also leave a comment explaining why you didn't change these to use `action` functions

Reply:
Done.
I think that the block 
`if (elementIsMasked) {
       await browser.executeScript('arguments[0].click()', await clickableElement.getWebElement());` 
which was there to fix the `element click intercepted errors`  in `action.js` was not working as expected in this case. But I am not sure about that. And after I removed the action statements, the test was passing!